tabs-tabs-tabs/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "applications": {
+    "gecko": {
+      "id": "tabs-tabs-tabs@mozilla.org",
+      "strict_min_version": "47.0.0"
```

Comment:
This doesn't install in my Nightly, I guess because it does not match 47.0a1. "47.*" doesn't, either (I guess "*" is interpreted as numeric (?)).

"47.0a1" works. Would it be OK to use that?